internal/state/state.go

```diff
@@ -18,8 +18,8 @@ import (
 //-----------------------------------------------------------------------------
 
 type Version struct {
-	Consensus version.Consensus ` json:"consensus"`
-	Software  string            ` json:"software"`
+	Consensus         version.Consensus ` json:"consensus"`
+	TendermintVersion string            ` json:"tendermint_version"`
```

Comment:
could rename it to TmVersion or something else if there is a preference

Reply:
Is there some reason "Tendermint" needs a "Version" suffix when "Consensus" does not?  If not, I think they should be symmetric.  I would probably leave off the suffix, but YMMV.

Changing the JSON key makes this user-visible, though I don't know offhand whether we encode these in RPC responses. If we do, is that going to cause problems for anyone with existing tools?

Reply:
I think `_version` is redundant, but I think the json tags on this are dodgy at best (because it should be in types if we put it on RPC? and why are we serializing external types?